shell/src/main/java/org/apache/accumulo/shell/commands/GrepCommand.java

```diff
@@ -49,6 +49,20 @@ public int execute(final String fullCommand, final CommandLine cl, final Shell s
       if (cl.getArgList().isEmpty()) {
         throw new MissingArgumentException("No terms specified");
       }
+      // Configure formatting options
+      final FormatterConfig config = new FormatterConfig();
+      config.setPrintTimestamps(cl.hasOption(timestampOpt.getOpt()));
```

Comment:
@rsingh433 I just noticed that this print timestamps feature is already enabled on line 104, so this is redundant here.